### PR TITLE
Add Backend

### PR DESCRIPTION
**Pull request by Shaardeep**
Facing problem after add one user backend giving error can look at it.

### CODE REVIEW

**Comment by PatelNeelMahesh** [comment]
Seems you have updated node_modules kindly delete it and then we can review.

**Comment by Shaardeep** [comment]
Ok sir

**Comment by Shaardeep** [comment]
Sir I have Delete the node_module file

File: Backend/index.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const ex = require("express");
+const mongoose = require("mongoose");
+const cors = require("cors");
+const UserModel = require("./models/Users");
+
+const app = ex();
+app.use(ex.json());
+app.use(cors());
+
+mongoose.connect("mongodb://127.0.0.1:27017/TCS");
```
Comment (PatelNeelMahesh) [inline Backend/index.js:10]: Note: when you put MongoDB Atlas link here do add it in an env variable.

**Review by PatelNeelMahesh — APPROVED** [review]
You can merge pull request.

**Pull request merged**